docs/glossary.md

```diff
@@ -0,0 +1,203 @@
+[N](#glossn)
+[O](#glosso)
+[P](#glossp)
+[Q](#glossq)
+[R](#glossr)
+[S](#glosss)
+[T](#glosst)
+[U](#glossu)
+[V](#glossv)
+[W](#glossw)
+[Z](#glossz)
+
+<!--end letter link tags-->
+
+## A
+{: #glossa}
+
+
+### AWT font subsystem
+{: #x10447151}
+
+The Abstract Window Toolkit (AWT) package in Java that is used to create graphical user interface applications. It contains classes that help to implement common windows-based tasks, such as adding scroll bars, buttons, list items, and text boxes.
+
+## C
+{: #glossc}
+
+
+### cryptographic acclerator
+{: #x10447156}
+
+The use of hardware to run cryptographic operations to improve performance.
+
+## D
+{: #glossd}
+
+
+### dump
+{: #x2021514}
+
+A capture of storage information at the time of an error.
+
+### dynamic logical partitioning
+{: #x10447168}
+
+Partitioning with the capability to logically attach and detach a managed system's resources to and from a logical partition's operating system without rebooting.
+
+## H
+{: #glossh}
+
+
+### heap dump
+{: #x2788665}
+
+A dump that shows the content of the Java heap.
+
+### heuristic
+{: #x2085115}
+
+A problem-solving method or algorithm used to derive a workable solution in within a reasonable time.
+
+### huge page
+{: #x4550512}
+
+In Linux, a block of memory that can be 2 MB or 1 GB in size. Using huge pages increases the kernel-level performance by creating fewer pages and therefore fewer mapping tables that the kernel has to load.
+
+## J
+{: #glossj}
+
+
+### Java dump
+{: #x8172135}
+
+A dump that contains information from the Java application, such as the operating environment, locks, threads, hooks, shared classes, and class loaders.
+
+## L
+{: #glossl}
+
+
+### large page
+{: #x10447162}
+
+In Windows, a block of memory that can be 2 MB or 1 GB in size.
+
+### logical partition (LP,LPAR)
+{: #x2002959}
+
+A function of an operating system that creates segments of resources that can be run on copies, or instances, of the operating system and associated applications.
+
+### LP
+{: #x2029334}
+See [logical partition](#x2002959).
+
+
+### LPAR
+{: #x2029338}
+See [logical partition](#x2002959).
+
+
+## O
+{: #glosso}
+
+
+### OpenSSL
+{: #x10447142}
+
+An open source version of the SSL and TLS security protocols, which provide encryption and server authentication over the internet.
+
+## P
+{: #glossp}
+
+
+### page
+{: #x2003565}
+
+A block of memory that equals 4096 bytes.
+
+## R
+{: #glossr}
+
+
+### release cadence
+{: #x10447137}
+
+The frequency at which new software updates or features are released to users.
+
+## S
+{: #glosss}
+
+
+### SDUMP
+{: #x2685105}
+See [system dump](#x2095425).
+
+
+### Secure Sockets Layer (SSL)
+{: #x2038004}
+
+A security protocol that provides communication privacy. With SSL, client/server applications can communicate in a way that is designed to prevent eavesdropping, tampering, and message forgery.
+
+### SSL
+{: #x2483907}
+See [Secure Sockets Layer](#x2038004).
+
+
+### system dump (SDUMP)
```

Comment:
Where does SDUMP come from, does OpenJ9 use this term?

Reply:
We are expected to use the Terminology-in-ibm for creating glossaries. The idea is to use standardised terms and definition across IBM. We can suggest changes and make it specific to each product. SDUMP might be an existing term that is being reused. I am coordinating with Carla Quinn. I will pass on these comments to her and work on correcting it. Please do let me know any other concerns you might have. Thanks!